app/models/availability.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
```

Comment:
Oops I overlooked something very important - add a unique constraint of the (user_id, day, time_range_id) tuple! (Figure out how to add a migration to add a unique index)